beacon-chain/core/altair/reward_test.go

```diff
@@ -195,7 +195,7 @@ func Test_BaseRewardPerIncrement(t *testing.T) {
 			errString:     "",
 		},
 		{
-			name:          "active balance is 256eth * 1m validators",
+			name:          "active balance is 256eth * 1b validators",
```

Comment:
```suggestion
			name:          "active balance is 256over * 1b validators",
```
nit: isn't this over? or is the unit still eth?

Reply:
It's Over.